modules/cluster-logging-collector-syslog.adoc

```diff
@@ -0,0 +1,184 @@
+</store>
+----
+
+<1> The Fluentd *syslog* plug-in.
+<2> The FQDN or IP address of the syslog server.
+<3> The port number to connect on. Defaults to 514.
+<4> The name of the syslog server.
+<5> Removes the prefix from the tag. Defaults to '' (empty).
+<6> The field to set the syslog key.
+<7> The syslog log facility or source.
+<8> The syslog log severity.
+<9> Determines whether to use the severity and facility from the record if available.
+<10> The key to look on the record to set the payload on the syslog message. Defaults to `message`.
+
+
+// Above definitions from https://github.com/docebo/fluent-plugin-remote-syslog
+
+
+.Sample `syslog` ConfigMap based on the configuration
+
+[source,yaml]
+----
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: syslog
+  namespace: openshift-logging
+data:
+  sysconfig.conf: |
+    <store>
+     @type syslog_buffered
+     remote_syslog syslogserver.openshift-logging.svc.cluster.local
+     port 514
+     hostname fluentd-4nzfz
+     remove_tag_prefix tag
+     tag_key ident,systemd.u.SYSLOG_IDENTIFIER
+     facility local0
+     severity info
+     use_record true
+     payload_key message
+    </store> 
+----
+
+.Procedure
+
+To configure the Fluentd *syslog* plug-in:
+
+. Create a configuration file named `sysconfig.conf` that contains the following
+parameters within the `<store>` stanza: 
+
+.. Specify the Fluentd *syslog* plug-in type:
++
+----
+@type syslog_buffered <1>
+----
++
+<1> Specify the plug-in to use, either: `syslog` or `syslog_buffered`. 
+
+.. Configure the name, host, and port for your external syslog server:
++
+[source,yaml]
+----
+remote_syslog <fqdn or IP> <1>
+port <number> <2>
+hostname <name> <3>
+----
++
+<1> Specify the FQDN or IP address of the syslog server.
+<2> Specify the port of the syslog server.
+<3> Specify a name for this syslog server.
++
+For example:
++
+[source,yaml]
+----
+remote_syslog syslogserver.openshift-logging.svc.cluster.local
+port 514
+hostname fluentd-server	
+----
++
+If you specify two or more receivers, the Fluentd *syslog* plug-in uses these servers in a round-robin order.
+
+.. Configure the other syslog variables as needed:
++
+----
+remove_tag_prefix <1>
+tag_key <key> <2>
+facility <value>  <3>
+severity <value>  <4>
+use_record <value> <5>
+payload_key message <6>
+----
++
+<1> Add this parameter to remove the `tag` field from the syslog prefix.
+<2> Specify the field to set the syslog key.
+<3> Specify the syslog log facility or source. For values, see link:https://tools.ietf.org/html/rfc3164#section-4.1.1[RTF 3164].
+<4> Specify the syslog log severity. For values, see link:link:https://tools.ietf.org/html/rfc3164#section-4.1.1[RTF 3164]. 
+<5> Specify `true` to use the severity and facility from the record if available. If `true` the `container_name`, `namespace_name`, and `pod_name` are included in the output content.
+<6> Use this field as the key to use to set the payload on the syslog message. Defaults to `message`.
```

Comment:
Or maybe just

_Specify the key to use to set the payload of the syslog message. ..._

And I think setting the payload _of_ makes more sense than _on_.